server/src/immich/api-v1/asset/asset.controller.ts

```diff
@@ -122,6 +122,7 @@ export class AssetController {
   @SharedLinkRoute()
   @Get('/file/:id')
   @Header('Cache-Control', 'private, max-age=86400, no-transform')
+  @ApiOkResponse({ content: { 'application/octet-stream': { schema: { type: 'string', format: 'binary' } } } })
```

Comment:
I added this decorator back for `serveFile` since `application/octet-stream` is in the OpenAPI spec for it and there's too many possible content types to narrow it down.